webapp/src/ts/app.component.ts

```diff
@@ -646,6 +650,23 @@ export class AppComponent implements OnInit, AfterViewInit {
     });
   }
 
+  private checkBrowserCompatibility(): void {
+    if (!this.isCompatibleChromeBrowser()) {
+      this.modalService.show(BrowserCompatibilityComponent);
+    }
+  }
+
+  private isCompatibleChromeBrowser(): boolean {
+    const browserInfo = this.browserDetectorService.detect();
+
+    if (browserInfo.name !== 'Chrome' || !browserInfo.version) {
+      return true;
+    }
+
+    const majorVersion = parseInt(browserInfo.version.split('.')[0]);
+    return !(majorVersion < 72 || (majorVersion >= 72 && majorVersion < 90));
```

Comment:
Please use constants for these versions numbers.

Can you [please confirm if it's 72 or 74](https://docs.communityhealthtoolkit.org/contribute/code/core/run-multiple-chrome-versions/)? As per [this ticket, it seems that it was failing in v74](https://github.com/medic/cht-core/issues/8456#issuecomment-1681321031)

Reply:
Since it's the same message for when `< 72` and` >=72 && < 90`. 
Then, can this be simplified to `if majorVersion < 90 then show modal`?  
Could you double-check with Michael or Nicole if one message for both scenarios is okay? You can also send a demo video to them so they have a better idea of the experience and provide better feedback.